hail/python/hail/docs/change_log.md

```diff
@@ -24,6 +24,28 @@ an earlier version of Hail to read files written in a later version.
 
 ---
 
+## Version 0.2.29
+
+Released 2019-11-17
+
+### Bug fixes
+- (hail#7229) Fixed `hl.maximal_independent_set` tie breaker functionality. See linked pull request for details.
```

Comment:
I think this "see for details" bit is implied, I think you can omit that sentence.